### PR TITLE
Gal 403/clinic info lsoa table

### PR DESCRIPTION
**Pull request by zainm97**
## Description
https://nhsd-jira.digital.nhs.uk/browse/GAL-403

This ticket handles the front end. It makes an API request to the backend on mount to get the list of LSOA and eligible population in 1 mile radius of the selected site. 

When the user selects a distance, a API call is made on update which gets the data with the expanded range. 

This ticket assumes that:
- UniqueLsoa table is populated with entries that include columns for average easting and northing. (Ticket 473 https://nhsd-jira.digital.nhs.uk/browse/GAL-473)
- And that the population database has an attribute for what LSOA code they reside in.

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: src/app/views/invitation_planning/InvitationPlanning.js
Comment (zainm97) [inline src/app/views/invitation_planning/InvitationPlanning.js:1]: just changed api calls on this page

File: src/app/views/clinic_summary/ClinicSummary.js
Comment (zainm97) [inline src/app/views/clinic_summary/ClinicSummary.js:1]: just changed api calls on this page

File: src/app/views/clinic_information/ClinicInformation.js [hunk 1]
```diff
@@ -276,6 +298,49 @@ class ClinicInformation extends Component {
             )
           });
       });
+
+    // TODO:Replace api id with latest api id from aws console until we get custom domain name set up
```
Comment (Amar-NHS) [inline src/app/views/clinic_information/ClinicInformation.js:302]: what is the purpose of axios call below for API again? - same axios call is done just above this. Also could not see "inputValue" being declared or constructed.

**Review by Amar-NHS — COMMENTED** [review]
just had 1 question in comments. Can't run it to check properly because of backend not being fully available.

**Review by Abdul-NHS — APPROVED** [review]

**Comment by Abdul-NHS** [comment]
Looks good, calculation for total available to invite is :+1

File: src/app/views/clinic_information/ClinicInformation.js (hunk 1 above)
Comment (zainm97) [reply src/app/views/clinic_information/ClinicInformation.js]: removed

**Review by Amar-NHS — APPROVED** [review]
V.Noice!

**Review by Abdul-NHS — APPROVED** [review]

**Review by Amar-NHS — APPROVED** [review]
Looks fine.

**Pull request merged**